src/main.js

```diff
@@ -11,6 +11,7 @@ import "./css/index.scss";
 register("en", () => import(/* webpackChunkName: "en" */ "./lang/en.json"));
 register("de", () => import(/* webpackChunkName: "de" */ "./lang/de.json"));
 register("es", () => import(/* webpackChunkName: "es" */ "./lang/es.json"));
+register("pt", () => import(/* webpackChunkName: "es" */ "./lang/pt.json"));
```

Comment:
Olà @marlonemmerich, I don't know if it is a problem but you should rename ```/* webpackChunkName: "es" */``` with ```/* webpackChunkName: "pt" */ ```.
A side question, which portuguese version are you using: pt-BR or pt-PT?

Reply:
Hello! Omg, thanks for your suggestion! You're right! I will change this here
I'm using pt-BR. Maybe we can split in pt-BR and pt-PT
@TorstenDittmann , what you think about it? Maybe can we have pt-BR (Brazilian portuguese) and pt-PT (Portuguese Brazilian)?

Cya!

Reply:
Sounds reasonable 🤔 Let me verify really quick, iirc there was something blocking this 👍

Reply:
I will resolve this for now and merge it, since I can't remember what was blocking it 😄 Gonna leave portugese as it is for now and adapt #85 later that day.